src/olympia/migrations/891-remove-compat-app.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `compat_totals` DROP COLUMN `app`;
+DELETE FROM `compat_totals`;
```

Comment:
We have to (it's only a cache from the last time we ran the command anyway) because other we have 5 rows of total, and we only care about one of them. I could have kept the one we care about (Firefox's) but since the data in ES is going to be completely wrong and we'll need a reindex to be able to use the tool anyway, it didn't matter.